secretariat/admin.py

```diff
@@ -13,12 +13,41 @@ class UserAdmin(admin.ModelAdmin):
         "last_name",
         "is_active",
         "is_staff",
+        "is_outline_synchronized",
     )
     readonly_fields = ["outline_uuid"]
+    fieldsets = (
+        (
+            "Utilisateur/rice",
+            {
+                "fields": ("username", "password", "email", "first_name", "last_name"),
+            },
+        ),
+        (
+            "Synchronisation",
+            {
+                "fields": ("outline_uuid", "date_joined"),
+            },
+        ),
+        (
+            "Django",
+            {
+                "fields": ("is_active", "is_staff", "is_superuser"),
+            },
+        ),
+    )
+
+    @admin.display(description="Synchro Outline", boolean=True)
+    def is_outline_synchronized(self, obj):
+        return obj.outline_uuid is not None
```

Comment:
Ça donne des faux positifs sur les objets qui pourraient avoir un uuid "not a valid uuid". Je me dis que ce sont des cas à la marge que l'on va régler par ailleurs donc je n'y attache pas d'importance.